### PR TITLE
fixed queries or devices

### PR DESCRIPTION
**Pull request by sharmaneil**
## Bugs fixed in this:

- the database didn't have any of the default devices in them. Now it does. Note: the backend should probably be refactored to load this in via a script from an admin panel
- the UI prevents users from adding the same device multiple times, which maybe triggered a violation of unique values error on the server

### CODE REVIEW

**Comment by sharmaneil** [comment]
Not sure if the backend smoke tests are failing because of something I did in this PR; seems unrelated. 

This should get merged in

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java [hunk 1]
```diff
@@ -14,12 +19,35 @@
...
-    public OrganizationResolver(OrganizationService os, ApiUserService users) {
+    public OrganizationResolver(OrganizationService os, ApiUserService users, DeviceTypeService devices) {
         _userService = users;
+        _deviceService = devices;
     }
 
     public Organization getOrganization() {
+
+        List<DeviceType> currentDevices = _deviceService.fetchDeviceTypes();
+        ArrayList<String> currentDeviceNames = new ArrayList<String>();
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java:32]: This hurts me to read, just a little. It's fine, though, really. 😁 

```suggestion
        List<String> currentDeviceNames = currentDevices.stream().map(d->d.getName()).collect(Collectors.toList());
```

**Review by benwarfield-usds — APPROVED** [review]
Java parts look fine to me, at least on an interim basis.

Takeaways:
1. we should have a mechanism for preloading known-good devices, since that's static data. Could use a migration; would like not to.
2. I wanted to know if there was an external ID other than the name for a device—that's something we should probably find out (maybe from the Data Hub team?) ASAP, since if there is we might need to report it.

**Comment by benwarfield-usds** [comment]
Oh, and you should merge up or rebase to make sure tests pass.

**Comment by sharmaneil** [comment]
We could also have an admin page for whoever is managing this app (probably a non-engineer) to log in and add/remove orgs, update global constants like devices, do account management chores, etc

**Comment by benwarfield-usds** [comment]
We definitely should have that, yes!

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java (hunk 1 above)
Comment (sharmaneil) [reply backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java]: lol look at all that fancy function chaining and java-ing!

**Comment by sharmaneil** [comment]
okay im just going to merge this; i fixed two of the lint problems but i need to figure out how to update vscode to catch these warnings; right now i just make a single fix, push out the change, and then wait for the smoke test to fail with another error message.

Thanks ben for the review!

**Pull request merged**